index.rst

```diff
@@ -21,301 +21,433 @@ Some documentation on this system can be accessed on `Docushare Collection-11943
-       :alt: optics_top
-       :scale: 15 %
-
-#. Install glass window in shutter
-#. Assemble shutter
-#. Install limit switches
-
-    .. figure:: /_static/open_shutter.png
-       :name: open_shutter
-       :target: ../_images/open_shutter.png
-       :alt: open_shutter
-       :scale: 15 %
-
-    .. figure:: /_static/shutter_open_2.png
-       :name: shutter_open2
-       :target: ../_images/shutter_open_2.png
-       :alt: shutter_open2
-       :scale: 15 %
-
-    .. figure:: /_static/shutter_close.png
-       :name: shutter_close
-       :target: ../_images/shutter_close.png
-       :alt: shutter_close
-       :scale: 15 %
-
-    .. figure:: /_static/shutter_close_2.png
-       :name: shutter_close2
-       :target: ../_images/shutter_close_2.png
-       :alt: shutter_close2
-       :scale: 10 %
+Opening Box
+-----------
+
+You may be tempted to open the box because there are handles on it and you want to look inside. 
+If you do so without following these instructions, you will likely damage something. 
+You can open the Left door at any time without damage. 
+This is a good option for inspecting or adjusting the optics. 
+If you want to open the center or right doors, following these steps:
+
+#. Undo the wires to the "open" limit switch
+
+      .. figure:: /_static/step_1.jpg
+         :name: step_1
+         :target: ../_images/step_1.jpg
+         :alt: step_1
+
+#. Remove the pin and nut from the linear actuator
+
+      .. figure:: /_static/step_2.jpg
+         :name: step_2
+         :target: ../_images/step_2.jpg
+         :alt: step_2
+
+#. Remove the linear actuator and place it out of the way
+
+      .. figure:: /_static/step_3.jpg
+         :name: step_3
+         :target: ../_images/step_3.jpg
+         :alt: step_3
+
+#. Remove the final bolt on the shutter and remove the shutter
 
-#. Open slits on monochromator
-#. Confirm that remote control is selected on WLS
+      .. figure:: /_static/step_4.jpg
+         :name: step_4
+         :target: ../_images/step_4.jpg
+         :alt: step_4
 
-Now you are ready to start turning things on and testing things. 
+#. If you need to open the center door, remove the "closed" limit switch
 
-Operational Setup
------------------
-First, addresses for all of the control components will have to be established. That is being tracked in JIRA ticket `IT-4096 <https://jira.lsstcorp.org/browse/IT-4096>`__. 
+      .. figure:: /_static/step_5.jpg
+         :name: step_5
+         :target: ../_images/step_5.jpg
+         :scale: 50 %
 
-Make sure that the indexes for each item is correct (for the CSC) and override yaml files are defined and identified.
 
-Then, go to the address for the PDU and confirm that the numbering of the outlets agrees with the table above and what is actually implemented.
+Chiller Maintenance
+-------------------
+
+You may need to refill the water in the chiller occassionally. 
+This should be done with distilled water mixed with 10% of methanol.
```

Comment:
If you know how much fluid it takes off the top of your head it might be nice to add it... even if it's an approximation.